prob/torus_cbc/build.py

```diff
@@ -146,12 +148,31 @@
 if USE_TABLE:
     bhl.report_var('Disk Ye',YE)
 
+if USE_TABLE:
+    EOS_TYPE = "EOS_TYPE_TABLE"
+elif DO_GAMMA_RADPRESS:
+    EOS_TYPE = "EOS_TYPE_GAMMA_RADPRESS"
+elif DO_GAMMA_GASPRESS:
+    EOS_TYPE = "EOS_TYPE_GAMMA_GASPRESS"
```

Comment:
now there should be a throw if it's not one a valid EOS type. Let's catch that in case someone somehow doesn't set one.